recipes/pantools/pantools.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/opt/anaconda1anaconda2anaconda3/bin/python
```

Comment:
Hmm. I see why you'd want to do this, but I'm not sure about the `anaconda1...` string. Is that one documented somewhere, or does it just happen to work at the moment?

Reply:
It does work, and in https://readthedocs.com/projects/continuumio-conda/downloads/pdf/latest/ `/opt/anaconda1anaconda2anaconda3` is still used as the default placeholder...

Reply:
Ok. Looks documented then.